addons/calendar/i18n/calendar.pot

```diff
@@ -269,21 +183,7 @@ msgid ""
 "    <p>\n"
 "        Hello <t t-out=\"object.common_name or ''\">Wood Corner</t>,<br><br>\n"
 "\n"
-"        <t t-if=\"is_online and target_customer\">\n"
-"            Your appointment <strong t-out=\"object.event_id.appointment_type_id.name or ''\">Schedule a Demo</strong> <t t-if=\"object.event_id.appointment_type_id.category != 'custom'\"> with <t t-out=\"object.event_id.user_id.name or ''\">Ready Mat</t></t> has been booked.\n"
-"            <t t-if=\"is_online and object.state != 'accepted' and object.event_id.appointment_type_id.resource_manual_confirmation\">\n"
-"                You will receive a mail of confirmation with more details when your appointment will be confirmed.\n"
```

Comment:
Hello @AH-Yussef @nle-odoo Beside updating the translation, one problem that i have encountered is error when sending email template in different language because the translation value in other language is really outdated. For example, if you look into this, `my custom addons` `Appointment (different from odoo enterprise appointment)` does not have `resource_manual_confirmatio.n` while odoo enterprise has then i will get error just because we have outdated translation